### DISCUSSION TITLE
tokenizer chat_template中"\\n"和"\n"是有差异么？

### DISCUSSION THREAD

**Discussion by NICEMXM** [01-ai/Yi#545] [Q&A]
下面第一张图是huggingface中权重tokenizer配置文件chat_template如下所示，模版中为"\\n"；第二张是transformers加载tokenizer的chat_template，模版中为"\n"，这有区别么，是否有影响？
![image](https://github.com/user-attachments/assets/2e7101ea-cce4-41c3-a2cd-84571d21fec6)
![image](https://github.com/user-attachments/assets/fbbdbd2f-ebc0-483d-8533-4be0508b5270)

**Comment by Haijian06** [comment]
@NICEMXM 感谢提问，这个是没有影响的，您现在使用起来是否有相关的疑问呢？
